### PR TITLE
feat: add Persian localization

### PR DESCRIPTION
**Pull request by sdtaheri**
Please let me see the results in the app if possible. Some texts may need changing.

### CODE REVIEW

**Comment by steventroughtonsmith** [comment]
Sure thing, give me a few to take some pictures

File: fa.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "از حذف این دسته و همه ایستگاه‌های داخل آن اطمینان دارید؟";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "حذف \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "آیا از حذف این دسته اطمینان دارید؟";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "بدون ایستگاه";
+
+"WELCOME_COPY" = "به Broadcasts خوش آمدید. اگر مایل باشید، ما چند ایستگاه رادیویی از محل زندگی خودمان به شما معرفی می‌کنیم تا کتابخانه‌تان را با آن آغاز کنید.\n\nیا اینکه اگر ترجیح می‌دهید به نسخه غیررایگان به روز رسانی کنید تا بتوانید ایستگاه‌های دلخواه خود را اضافه کنید یا از بین هزاران ایستگاه در فهرست نرم‌افزار انتخاب کنید.";
+
+"WELCOME_UPSELL" = “به Broadcasts خوش آمدید. اگر مایل باشید، ما چند ایستگاه رادیویی از محل زندگی خودمان به شما معرفی می‌کنیم تا کتابخانه‌تان را با آن آغاز کنید.\n\nیا اینکه اگر ترجیح می‌دهید به نسخه غیررایگان به روز رسانی کنید تا بتوانید ایستگاه‌های دلخواه خود را اضافه کنید یا از بین هزاران ایستگاه در فهرست نرم‌افزار انتخاب کنید.”;
```
Comment (steventroughtonsmith) [inline fa.lproj/Localizable.strings:87]: Outer quotes need adjusting here back to regular `"`

**Comment by steventroughtonsmith** [comment]
How does this look?

Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 04 41
Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 04 45
Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 04 54
Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 05 00
Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 05 05
Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 18 05 09

File: fa.lproj/Localizable.strings (hunk 1 above)
Comment (sdtaheri) [reply fa.lproj/Localizable.strings:87]: Sure thing. That was iPad’s and Working Copy’s fault.

**Comment by sdtaheri** [comment]
They look great.

**Comment by steventroughtonsmith** [comment]
Awesome, thanks! Can I credit you as "Saeed Taheri"?

**Comment by sdtaheri** [comment]
I would be honored. Thanks a lot.

**Pull request merged**